db/migrate/20161214033216_enable_uuid_extension.rb

```diff
@@ -0,0 +1,11 @@
+class EnableUuidExtension < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp'
```

Comment:
I'm glad that I started "correct", although I think that avoiding a puppet workaround is best. I'll implement your suggestion here.